README.rst

```diff
@@ -93,7 +97,7 @@ and run the following ::
 
 This package also provides command-line utilities. To check the software has been installed correctly, type ::
 
-        diffpy.srxplanar --version
+        diffpy-srxplanar --version
```

Comment:
I think the command line app can just be called srxplanar.  Less for folks to type